modules/oadp-mcg-performance-tuning.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-mcg-performance-tuning_{context}"]
+= Performance tuning guide for Multicloud Object Gateway
+
+The default configuration of {rh-storage} for Multicloud Object Gateway is optimized for low resource consumption and not performance.
```

Comment:
Sorry, IMO it's not clear what this module is trying to explain. Is there any other suitable place where this information can be added?

Reply:
So originally I had pulled across the help article and included it as procedure

https://issues.redhat.com/browse/OADP-3737

![image](https://github.com/openshift/openshift-docs/assets/106804821/0c19b64f-9c86-400c-afe6-d2288bcd8ff4)

Reply:
Well, IMO, this is not ready to merge. I see some gaps. I suggest to: 

- Confirm if a separate module is required or not.
- Check if a different heading can be given other than "Troubeshooting....." (check with SME/QE perhaps for accuracy)
- Check if there is other way/place to present this information (KCS article)
- Can this KCS article be linked under additional resources?